packages/charts/react-charts/library/src/components/HeatMapChart/HeatMapChart.tsx

```diff
@@ -0,0 +1,657 @@
+import * as React from 'react';
+import { HeatMapChartProps } from './HeatMapChart.types';
+import { AccessibilityProps, Chart, HeatMapChartData, HeatMapChartDataPoint, Margins } from '../../types/index';
+import {
+  ChartTypes,
+  convertToLocaleString,
+  getAccessibleDataObject,
+  getColorContrast,
+  getTypeOfAxis,
+  resolveCSSVariables,
+  XAxisTypes,
+  YAxisType,
+} from '../../utilities/index';
+import { CartesianChart, ChildProps } from '../CommonComponents/index';
+import { useId } from '@fluentui/react-utilities';
+import { tokens } from '@fluentui/react-theme';
+import { useHeatMapChartStyles } from './useHeatMapChartStyles.styles';
+import { Legend, Legends } from '../Legends/index';
+import { scaleLinear as d3ScaleLinear } from 'd3-scale';
+import { format as d3Format } from 'd3-format';
+import { timeFormat as d3TimeFormat } from 'd3-time-format';
+
+type DataSet = {
```

Comment:
>DataSet [](http://example.com/codeflow?start=5&length=7)

Issue observed.
The callout is becoming too large if description message is long

Reply:
Will have to modify ChartPopover to accept maxWidth as a prop. Can I follow up on this later?

Reply:
lets log a bug for this